src/server/PostManager.js

```diff
@@ -1,6 +1,34 @@
+import db from '../server/database';
+import {KeyUtils, ENTITY} from '../server/key-utils';
 
 export default class PostManager {
```

Comment:
this function is not only for item. image can be added as created post too.
I think parameter 'item' is bad naming. i will modify it.